packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
   * Added example for `terra-menu` with disabled item.
   * Added a `terra-slide-panel` test containing multiple slide panels.
   * Added testing sections to `terra-menu`, `terra-navigation-prompt`, `terra-pills` and `terra-tabs` documentation.
+  * Added documentation and tests for `terra-worklist-data-grid`.
+  * Added documentation and tests for additional keyboard navigation in `terra-worklist-data-grid`.
```

Comment:
Looks like these are under the June 5th release, probably from merging? They should be moved to the Unreleased section.

Reply:
This file was incorrectly merged. Fixed along with @sdadn observation below.

Reply:
Looks like these are still under the wrong section, can you move them to Unreleased?